el_hub/el_hub_flutter/lib/src/screens/home_screen.dart

```diff
@@ -1,18 +1,38 @@
+import 'package:el_hub_client/el_hub_client.dart';
+import 'package:el_hub_flutter/bloc/power_installations_bloc.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
 
 import '../../main.dart';
+import '../widgets/home/home_header.dart';
 
-class HomeScreen extends StatelessWidget {
+class HomeScreen extends StatefulWidget {
   const HomeScreen({super.key});
 
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
   @override
   Widget build(BuildContext context) {
+    
+        
     return Scaffold(
-      appBar: AppBar(title: const Text("El Hub")),
+        appBar: AppBar(title: const Text("El Hub")),
         body: Center(
-          child: Row(children: [
-            Text("Velkommen ${sessionManager.signedInUser?.userName}"),
+          child:
+            Column(crossAxisAlignment: CrossAxisAlignment.center, children: const [
+            HomeHeaderWidget(),
```

Comment:
Cool that you insert the welcome text for users in a widget since it is being used all over the site when the user is logged in :)